OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/ResetPasswordDialogFragment.java

```diff
@@ -60,7 +60,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                 null, false);
```

Comment:
Please fix the other warnings as well

Reply:
I've removed the warnings that won't require logic change.